nexusadspy/segment.py

```diff
@@ -0,0 +1,136 @@
+
+from __future__ import (
+    print_function, division, generators,
+    absolute_import, unicode_literals
+)
+
+from nexusadspy.client import AppnexusClient
+from io import BytesIO
+from gzip import GzipFile
+
+
+class MemberOfSegment:
+    """
+    Class to hold attributes about the user to be added in segment.
+    """
+    def __init__(self,appnexus_user_id, timestamp, mobile_os=None, expiration=0, numeric_value=0):
+        """
+        Detailed documentation about values can be found here:
+        https://wiki.appnexus.com/display/api/Batch+Segment+Service+-+File+Format
+        :param appnexus_user_id: str, Appnexus ID of the user. IDFA/AAID in case of mobiles.
+        :param timestamp: int, Unix timestamp at which user goes into effect in segment.
+        :param mobile_os: str, mobile OS used by user. None if desktop.
+        :param expiration: int, Lifetime of user in segment (e.g. android, ios). 0 means never expire.
+        :param numeric_value: int, Numeric value for the user.
+        """
+        self._user_id = appnexus_user_id
+        self._expiration = expiration
+        self._timestamp = timestamp
+        self._numeric_value = numeric_value
+        self._mobile_os = mobile_os
+
+    def get_user_id(self):
+        return self._user_id
+
+    def get_timestamp(self):
+        return self._timestamp
+
+    def get_expiration(self):
+        return self._expiration
+
+    def get_value(self):
+        return self._numeric_value
+
+    def get_mobile_os(self):
+        return self._mobile_os
+
+
+class SegmentsUploaderJob:
```

Comment:
please follow the naming convention in AppnexusReport https://github.com/mathemads/nexusadspy/blob/master/nexusadspy/report.py